lib/elixir/lib/calendar/time.ex

```diff
@@ -656,12 +666,9 @@ defmodule Time do
   additional information about a date or time zone is ignored when calculating
   the difference.
 
-  The answer can be returned in any `unit` available from
-  `t:System.time_unit/0`. If the first time value is earlier than
-  the second, a negative number is returned.
-
-  This function returns the difference in seconds where seconds
-  are measured according to `Calendar.ISO`.
+  If the first time value is earlier than the second, a negative number is
+  returned. The answer can be returned in any `unit` available from
+  `t:time_unit/0`.
```

Comment:
These two sections seemed contradictory so I removed the second segment. Looking at it now I realize that there isn't any mention of `Calendar.ISO` units, so I need to put that back.